docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 aenum
-apache-airflow==2.10.2
+apache-airflow==2.10
```

Comment:
@pankajkoti WDYT
```suggestion
apache-airflow~=2.10.3
```

Reply:
yes, that sounds perfect. I guess the same effect as `apache-airflow>=2.10.3,<2.11` right?

Reply:
Yes, just using less characters 😄